sitzverteilung-frontend/src/views/DataView.vue

```diff
@@ -19,9 +19,96 @@
       @no="hideDeleteConfirmation"
       @yes="deleteSelectedBaseData"
     />
-    <v-row>
-      <v-col>
-        <h1>Verwaltung der Basisdaten</h1>
+    <v-row
+      align="center"
+      class="d-flex"
+    >
+      <v-col class="d-flex align-center">
+        <h1 class="mr-2">Verwaltung der Basisdaten</h1>
+        <info-dialog>
+          <template #dialog-text>
+            <div class="dialog-text pa-4">
+              <h3 class="mb-2">Information zur Basisdatenübersicht</h3>
```

Comment:
I'd not use h3 and h4 elements here as it does not fit into the heading structure of the rest of the page (headings should always be h1 -> h2 -> h3 -> ... and no layer should be skipped).
Thus you should correct the heading levels according to the rest of the heading levels in the view. (I guess it should be h2 as first heading inside the dialog because we use h1 at line 27)
If you then want to style the headings differently, use the typography helper classes of Vuetify.
```suggestion
              <h2 class="mb-2">Information zur Basisdatenübersicht</h2>
```